docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    mp_dist = stumpy.mpdist(t_1, t_2, 30, percentage=0.05)
</pre><p>Might consider storing <code>m = 30</code>  and <code>percentage = 0.05</code>  since you use this multiple times.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Hmm, although the values are same, they are used in two completely different contexts so I don't think they necessarily need to be the same. Moreover, it's probably more easier to see what the parameters are instead of having to refer back if the reader forgets. </p>

Reply:
<p>Oh true. Maybe you could do <code>m = 30</code>  twice in two different code blocks, for the sake of cleanliness. Hardcoded values are often confusing to me without context.</p>